sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample10_ClaimCheck.md

```diff
@@ -0,0 +1,56 @@
+## Claim check pattern
+
+This sample demonstrates the use of the [claim check pattern](https://docs.microsoft.com/azure/architecture/patterns/claim-check) which enables you to work with arbitrarily large message bodies. For standard namespaces, a message can be at most 256 KB. For Premium namespaces, the limit is 100 MB. If these limits don't work for your application you can leverage Azure Storage Blobs to implement this pattern.
+
+### Sending the message
+
+In our example, we will assume that the message body can fit in memory. This allows us to use the Storage Blob methods that let you work with `BinaryData`. If your message body cannot fit in memory, you can use the stream-based Upload/Download methods instead.
```

Comment:
Should we consider linking to the Blobs docs for the stream-based methods that you're referencing?